pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -72,13 +75,14 @@ type ReconcileAPIScheme struct {
 	scheme *runtime.Scheme
 }
 
-// Reconcile reads that state of the cluster for a APIScheme object and makes changes based on the state read
-// and what is in the APIScheme.Spec
-// TODO(user): Modify this Reconcile function to implement your Controller logic.  This example creates
-// a Pod as an example
-// Note:
-// The Controller will requeue the Request to be processed again if the returned error is non-nil or
-// Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
+// Reconcile will ensure that the rh-api management api endpoint is created and ready.
```

Comment:
for the code, (i know, it's a comment), let's not use the name 'rh-api'.  This should be configurable, even though we will exclusively use "rh-api"